### ISSUE TITLE
didnt work with express and ejs

### ISSUE THREAD

**Issue by nitplayz** [Glitchii/embedbuilder#22]
idk wtf is wrong with the code so that it wont show correctly on my bot dashboard?
it works fine when i open it with  live server
![Screenshot_2022-06-11_13-09-03](https://user-images.githubusercontent.com/83857989/173178607-2cad4d8d-6c34-4b5c-8952-e2e1223dfbd4.png)
![Screenshot_2022-06-11_13-09-37](https://user-images.githubusercontent.com/83857989/173178611-418c4ec3-e411-43e1-8426-964e21d93676.png)
![Screenshot_2022-06-11_13-10-15](https://user-images.githubusercontent.com/83857989/173178612-8ec810d7-f938-477b-939b-bdcd8c2fd414.png)
![Screenshot_2022-06-11_13-10-40](https://user-images.githubusercontent.com/83857989/173178613-e0a0c61e-a514-4461-8335-a35a18c974a6.png)

**Comment by HexM7** [comment]
Looks like the stylesheet isn't being loaded, can you check your browser console for any errors? Also check if the stylesheet is loaded by having a peek into the DevTool's network tab.

**Comment by Glitchii** [comment]
TLDR; You don't have styling because the CSS isn't loaded, as mentioned by HexM7. You should see an error in the console about that.

I see your path is `/dashboard/:guildID/embed`. On (or near) [line 16 in index.html](https://github.com/Glitchii/embedbuilder/blob/master/index.html#L16=), there's a HTML link like this 
```html
<link rel="stylesheet" href="./assets/css/index.css" />
```
Because your path is `/dashboard/:guildID/embed`, this link actually points to `/dashboard/:guildID/embed/assets/css/index.css`. It fails because the path `/dashboard/:guildID/embed/assets/css/index.css` does noo exist on your server.

You could fix this by either manually editing that line to add path to your CSS, or in Express, you can make a route for `/assets/css/index.css` that redirects (res.redirect) to your actual CSS file
